nestkernel/nest_time.h

```diff
@@ -473,6 +485,7 @@ class Time
   {
```

Comment:
This is where you're getting extra complexity that will also cost performance: anyplace where you have a loop around `time += n secs` you're going to compute unneeded steps. It's better to cache as in event.h, where you know you need to keep the value, and not everywhere in the code where, if you're lucky, it all balances out to zero, or gives you sometimes a gain, and sometimes a loss.